apps/meteor/app/autotranslate/server/autotranslate.ts

```diff
@@ -305,10 +305,13 @@ export abstract class AutoTranslate {
 			Meteor.defer(() => {
 				for (const [index, attachment] of message.attachments?.entries() ?? []) {
 					if (attachment.description || attachment.text) {
-						const translations = this._translateAttachmentDescriptions(attachment, targetLanguages);
+						// Removes the initial link `[ ](quoterl)` from quote message before translation
+						const translatedText = attachment?.text?.split(') ')[1] || attachment?.text;
```

Comment:
Nice improvement! But if I have more parenthesis `) ` like this it breaks the translation.
![image](https://user-images.githubusercontent.com/40830821/203401826-3f75c4d2-35c3-4ea3-8434-7e984c8a65f0.png)